etc/zones/db.asuc.org

```diff
@@ -29,9 +29,6 @@ www	IN CNAME	@
 ; Note to readers: all these www CNAMEs were imported when asuc.org
```

Comment:
bump the serial in this file (github won't let me comment that far up)

i don't think the other serial bumps matter